nebula-logger/core/tests/log-management/classes/LogEntryEventHandler_Tests.cls

```diff
@@ -836,7 +908,12 @@ private class LogEntryEventHandler_Tests {
         System.assertEquals(logEntryEvent.ParentLogTransactionId__c, log.ParentLogTransactionId__c, 'log.ParentLogTransactionId__c was not properly set');
         System.assertEquals(logEntryEvent.ProfileId__c, log.ProfileId__c, 'log.ProfileId__c was not properly set');
         System.assertEquals(logEntryEvent.ProfileName__c, log.ProfileName__c, 'log.ProfileName__c was not properly set');
-        System.assertEquals(logEntryEvent.Scenario__c, log.Scenario__c, 'log.Scenario__c was not properly set');
+        System.assertEquals(
+            logEntryEvent.Scenario__c?.left(Schema.LogScenario__c.Name.getDescribe().getLength()),
```

Comment:
what's the mechanism here that's setting the Scenario here? I didn't see anything in setup creating a scenario record, and the assert makes it seem like there could be truncation at play here, as well?

Reply:
Oh maybe it's just not part of the diff because I see you were previously asserting on the Scenario being set ...